keps/sig-network/0006-20180314-coredns-GA-proposal.md

```diff
@@ -0,0 +1,123 @@
+---
+kep-number: TBD
+title: Switch Default DNS to CoreDNS 
+authors:
+  - "@johnbelamaric"
+  - "@rajansandeep"
+owning-sig: sig-network
+participating-sigs:
+  - sig-cluster-lifecycle
+reviewers:
+  - "@bowei"
+  - "@thockin"
+approvers:
+  - "@thockin"
+editor: "@rajansandeep"
+creation-date: 2018-03-21
+last-updated: 2018-03-21
+status: provisional
+---
+ 
+# Switch Default DNS to CoreDNS 
+
+## Table of Contents
+
+* [Summary](#summary)
+* [Motivation](#motivation)
+    * [Goals](#goals)
+    * [Non-Goals](#non-goals)
+* [Proposal](#proposal)
+    * [User Cases](#use-cases)
+* [Unresolved Questions](#unresolved-questions)
+* [Graduation Criteria](#graduation-criteria)
+* [Implementation History](#implementation-history)
+
+## Summary
+
+CoreDNS is sister CNCF project and is the successor to SkyDNS, on which kube-dns is based. It is a flexible, extensible
+authoritative DNS server and directly integrates with the Kubernetes API. It can serve as cluster DNS,
```

Comment:
what does directly integrates with kubernetes API mean ? KubeDNS also does that right ?